src/mlpack/bindings/R/mlpack/src/rcpp_mlpack.h

```diff
@@ -15,9 +15,6 @@
 
 #include <Rcpp.h>
 
-// To suppress Found '__assert_fail', possibly from 'assert' (C).
-#define BOOST_DISABLE_ASSERTS
```

Comment:
@coatless @eddelbuettel @Yashwants19 if you don't mind sanity checking me on this one since I am not an R expert... I think it's no longer needed since we aren't using Boost.

Reply:
Agreed, just remove.

Reply:
Yup, no longer needed with the shift away from boost!